doc/build-helpers/trivial-build-helpers.chapter.md

```diff
@@ -92,6 +92,155 @@ writeShellScript "evaluate-my-file.sh" ''
 ```
 ::::
 
+### `makeDesktopItem` {#trivial-builder-makeDesktopItem}
+
+Write an [XDG desktop file](https://specifications.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html) to the Nix store.
+
+This function is most often used with `mkDerivation.desktopItems` to add desktop items to a package.
```

Comment:
Can you also add an example for how to use `makeDesktopItem` with `mkDerivation.desktopItems`? Can be just a reduced example taken from some package in Nixpkgs.

Reply:
I think it would be better to document the general case in the documentation for `stdenv.mkDerivation` rather than have `makeDesktopItem` be the only listed option, since an existing file can be used as well.

Reply:
I think ideally we would do both. I don't want to increase the scope of this PR to include adding content to `mkDerivation`'s docs, so that's why I asked to add the example here (and later we can reference the same example on `mkDerivation`'s docs).

If you don't want to add an example like this, feel free to skip this comment and I'll add when I do a pass on this part of the docs again.

Reply:
Looks like `mkDerivation` itself does nothing with `desktopItems`, rather it's done by having `copyDesktopItems` in `nativeBuildInputs`